README.md

```diff
@@ -1,55 +1,137 @@
 caliper-php
-================
+===========
 
-caliper-php is a php client for [Caliper](http://www.imsglobal.org) that provides an implementation of the Caliper SensorAPI™.
+caliper-php is a PHP client for [Caliper](http://www.imsglobal.org) that
+provides an implementation of the Caliper SensorAPI™.
 
 ## Getting Started
 
-### Pre-requisites for development:  
+### Pre-requisites for development
 
 * PHP 5.4 required (PHP 5.6 recommended)
 * Ensure you have php5 and php5-json installed:  ```sudo apt-get install php5 php5-json```
```

Comment:
I think this line is redundant as the before line says that PHP5.4 is required and php5-json is added to php from 5.2 version onwards